tensorflow/compiler/xla/service/gpu/gemm_rewriter.cc

```diff
@@ -501,9 +501,50 @@ class GemmRewriterVisitor : public DfsHloRewriteVisitor {
                       CublasLtMatmulMaybeF8(&existing_gemm).WithOneUser())
                       .WithOneUser(),
                   m::Broadcast(&bias, m::Op())))) {
-      TF_ASSIGN_OR_RETURN(
-          bool was_fused,
-          FuseVectorBiasAdd(instr, bias, existing_gemm, optional_slice));
+      HloInstruction *optional_bias_f16 = nullptr;
+      if (existing_gemm->custom_call_target() == kCublasLtMatmulF8CallTarget &&
```

Comment:
Do you still need the logic to handle the matrix bias in `CreateF8CustomCall`?

Reply:
Yes. The type has to be checked to ensure compatibility.